networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc

```diff
@@ -1,45 +1,74 @@
 :_mod-docs-content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
 [id="albo-sts-cluster"]
-= Preparing for the AWS Load Balancer Operator on a cluster using the AWS {sts-full}
+= Installing the AWS Load Balancer Operator on a cluster that uses AWS {sts-short}
```

Comment:
[In the preview](https://80881--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/installing-albo-sts-cluster) I still see the chapter called "Preparing ...". I think it can make sense to rename it to this title.